core/src/main/java/com/linecorp/armeria/server/CorsServerErrorHandler.java

```diff
@@ -0,0 +1,47 @@
+package com.linecorp.armeria.server;
+
+import com.linecorp.armeria.common.*;
+import com.linecorp.armeria.common.annotation.Nullable;
+import com.linecorp.armeria.internal.server.CorsHeaderUtil;
+import com.linecorp.armeria.server.cors.CorsService;
+
+public class CorsServerErrorHandler implements ServerErrorHandler {
+    ServerErrorHandler serverErrorHandler;
+
+    public CorsServerErrorHandler(ServerErrorHandler serverErrorHandler) {
+        this.serverErrorHandler = serverErrorHandler;
+    }
+
+    @Override
+    public @Nullable AggregatedHttpResponse renderStatus(@Nullable ServiceRequestContext ctx, ServiceConfig config, @Nullable RequestHeaders headers, HttpStatus status, @Nullable String description, @Nullable Throwable cause) {
+
+        CorsService corsService = config.service().as(CorsService.class);
+
+        if (corsService != null) {
+
+            AggregatedHttpResponse res = serverErrorHandler.renderStatus(ctx, config, headers, status, description, cause);
+
+            ResponseHeaders updatedResponseHeaders = addCorsHeaders(ctx, corsService, res.headers());
```

Comment:
Global comment: Could you add `final` keyword to every applicable variable declaration?

Reply:
Thank you for your comment.
I fixed those things.